app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/AboutFragment.kt

```diff
@@ -95,10 +95,26 @@ class AboutFragment : Fragment() {
                 "Apache License, Version 2.0",
             ),
             Library(
-                "AndroidX Preference eXtended",
+                "AndroidX Preference extended",
                 "https://github.com/takisoft/preferencex-android",
                 "Apache License, Version 2.0",
             ),
+            Library(
+                "KotlinX Coroutines",
+                "https://github.com/Kotlin/kotlinx.coroutines",
+                "Apache License, Version 2.0",
+            ),
+            Library(
+                "Kotlin Standard Library",
+                "https://kotlinlang.org/api/latest/jvm/stdlib/",
+                "Apache License, Version 2.0",
+            ),
+            Library("KotlinX Immutable Collections",
+                "https://github.com/Kotlin/kotlinx.collections.immutable",
+                "Apache License, Version 2.0"
+            ),
+            Library("KotlinX DateTime","https://github.com/Kotlin/kotlinx-datetime","Apache License, Version 2.0"),
+            Library("Kotlin Android Extensions Runtime","https://kotlinlang.org/","Apache License, Version 2.0"),
```

Comment:
Instead of including the URL of the Kotlin website, please link to the code: https://github.com/JetBrains/kotlin/tree/master/plugins/android-extensions/android-extensions-runtime